src/lib.rs

```diff
@@ -707,7 +692,7 @@ mod tests {
                 .unwrap()
         );
 
-        let triple = get_target_triple(Some("x86_64-pc-linux-gnu".into()));
+        let triple = get_target_triple(Some("x86_64-pc-linux-gnu"));
         assert_eq!(triple.as_str().to_str().unwrap(), "x86_64-pc-linux-gnu");
     }
 
```

Comment:
ill be honest, I didnt really check this file because its hard to see where all the code went and how this connects to the rest of the commit
i'll just gonna trust you here